--- a/axcl/drv/pcie/driver/axcl_host/axcl_pcie_host.c
+++ b/axcl/drv/pcie/driver/axcl_host/axcl_pcie_host.c
@@ -1561,4 +1561,8 @@
 
 MODULE_LICENSE("GPL");
 MODULE_AUTHOR("Axera");
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 13, 0)
+MODULE_IMPORT_NS("VFS_internal_I_am_really_a_filesystem_and_am_NOT_a_driver");
+#else
 MODULE_IMPORT_NS(VFS_internal_I_am_really_a_filesystem_and_am_NOT_a_driver);
+#endif
